frontend/src/js/tooltip/Tooltip.tsx

```diff
@@ -80,6 +83,10 @@ const Infos = styled("div")`
   overflow-x: auto;
 `;
 
+const IndentRoot = styled("div")<{ depth: number }>`
+  padding-left: ${({ depth }) => depth * 15 + "px"};
```

Comment:
Since you're only using depth=1, could you please just remove depth and just do
```suggestion
  padding-left: 15px;
```